src/database/models.py

```diff
@@ -1110,7 +1110,6 @@ def full_json(self):
         data = self.simple_json()
         # Q: should this be in simple_json?
         data["communities"] = [c.simple_json() for c in self.communities.all()]
-        # data["admins"] = [a.simple_json() for a in self.admins.all()]
```

Comment:
any reason we are removing this?

Reply:
Because it is a comment and the admins are returned by the simple_json method.